vunit/ui/package_facade.py

```diff
@@ -22,6 +22,7 @@ def __init__(self, parent, library_name, package_name, design_unit):
         self._library_name = library_name
         self._package_name = package_name
         self._design_unit = design_unit
+        self._source_file = self._design_unit.source_file
```

Comment:
Since `_source_file` is accessible through `source_file()`, is it really required to define it apart from `_design_unit`?